### PR TITLE
Added Edited submission Migration

### PR DESCRIPTION
**Pull request by ohyonghao**
Added the migration needed to use Edited Submissions.

### CODE REVIEW

File: Appserver/Controllers/TimesheetController.cs [hunk 1]
```diff
@@ -121,22 +121,53 @@ public IActionResult Submit(int id)
...
                 textractform.AddPages(childform);
             }
-            var tsf = (TimesheetForm)AbstractFormObject.FromTextract(textractform);
-
-            tsf.id = id;
+            var ts = (TimesheetForm)AbstractFormObject.FromTextract(textractform);
 
-            var dbutil = new FormToDbUtil(_subcontext, _context);
-            Timesheet ts = dbutil.PopulateTimesheet(tsf);
-            dbutil.PopulateTimesheetEntries(tsf, ts);
```
Comment (brialee) [inline Appserver/Controllers/TimesheetController.cs:130]: Should those db utilities be removed [from the IDD namespace] if they aren't being used any longer?

File: Appserver/Controllers/TimesheetController.cs [hunk 2]
```diff
@@ -121,22 +121,53 @@ public IActionResult Submit(int id)
...
+                });
+            }
 
-            var submission = _subcontext;
-            submission.Add(ts);
-            submission.SaveChanges();
-            
-            // Do something with form
-            Response.Headers.Add("Access-Control-Allow-Origin", "*");
-            Response.Headers.Add("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept");
```
Comment (brialee) [inline Appserver/Controllers/TimesheetController.cs:138]: I think these response headers might have been part of the issue when running the PWA locally?

File: Appserver/Controllers/TimesheetController.cs [hunk 3]
```diff
@@ -121,22 +121,53 @@ public IActionResult Submit(int id)
...
+            {
+                var val = new PWAsubmissionVals();
+                val.value = x;
+                val.wasEdited = false;
+                return val;
+            };
+
+            var PWAForm = new PWAsubmission();
+            PWAForm.approval = PWAConv(ts.approval.ToString());
+            PWAForm.authorization = PWAConv(ts.authorization.ToString());
```
Comment (iyves) [inline Appserver/Controllers/TimesheetController.cs:137]: Authorization (the checkbox at the very end of the form) is a Boolean, not a string.

Comment (iyves) [reply Appserver/Controllers/TimesheetController.cs:137]: Approval (the provider initials) _is_ a string, so approval's type is correct here.

File: Appserver/Controllers/TimesheetController.cs [hunk 4]
```diff
@@ -121,22 +121,53 @@ public IActionResult Submit(int id)
...
+            PWAForm.prime = PWAConv(ts.prime);
+            PWAForm.progressNotes = PWAConv(ts.progressNotes);
+            PWAForm.providerName = PWAConv(ts.providerName);
+            PWAForm.providerNum = PWAConv(ts.providerNum);
+            PWAForm.providerSignature = PWAConv(ts.employerSignature.ToString());
+            PWAForm.providerSignDate = PWAConv(ts.employerSignDate);
+            PWAForm.scpaName = PWAConv(ts.scpaName);
+            PWAForm.serviceAuthorized = PWAConv(ts.serviceAuthorized);
+            PWAForm.serviceGoal = PWAConv(ts.serviceGoal);
+            PWAForm.totalHours = PWAConv("20:00");
```
Comment (iyves) [inline Appserver/Controllers/TimesheetController.cs:152]: Should this be ts.totalHours rather than "20:00"?

**Pull request merged**